sdk/storage/storage-blob/src/generateSASUrl.ts

```diff
@@ -0,0 +1,22 @@
+import { BlobClient, GenerateSASUrlOptions } from "./BlobClient";
+import { SharedKeyCredential } from "./credentials/SharedKeyCredential";
+import { isNode } from "@azure/core-http";
+import { generateBlobSASQueryParameters } from "./BlobSASSignatureValues";
+
+BlobClient.prototype.generateSASUrl = function(
```

Comment:
What's reason to change prototype, instead of defining it in `BlobClient`?

Reply:
Removed.